planet4-gutenberg-blocks.php

```diff
@@ -207,7 +207,6 @@
 	'planet4-block-templates/quick-links',
 	'planet4-block-templates/reality-check',
 	'planet4-block-templates/issues',
-	'planet4-block-templates/page-header',
```

Comment:
I think this should stay for now? We didn't remove other blocks/patterns that were moved to the theme 🤔 Because of this, on the test instance you can't find the Page Header in the available blocks.